python/gui/MaaS/templates/maas/dataset_management.html

```diff
@@ -0,0 +1,797 @@
+            color: #ffc;
+            overflow: hidden;
+        }
+
+        #branding {
+            float: left;
+        }
+
+        #branding h1 {
+            color: #f5dd5d;
+            padding: 0px;
+            margin: 0 20px 0 0;
+            font-weight: 300;
+            font-size: 24px;
+        }
+
+        #container {
+            position: relative;
+            width: 100%;
+            min-width: 880px;
+            padding: 0;
+        }
+
+        body {
+            margin: 0;
+            padding: 0;
+            font-size: 14px;
+            font-family: Tahoma, Geneva, sans-serif;
+            color: #333;
+            background: #FFF;
+        }
+
+        #content {
+            padding: 20px 40px;
+        }
+
+        .message-list {
+            list-style-type: none;
+        }
+
+        #error-messages {
+            border: 1px solid black;
+            background-color: lightpink;
+        }
+
+        #warning-messages {
+            border: 1px solid black;
+            background-color: #FFC;
+        }
+
+        #info-messages {
+            border: 1px solid black;
+            background-color: palegreen;
+        }
+
+        #dataset-create-form-div-universal-inputs {
+            line-height: 25px;
+            max-width: 350px;
+        }
+
+        #dataset-create-form-div-dynamic-inputs {
+            line-height: 25px;
+            max-width: 350px;
+        }
+
+        .disabled {
+            color: gray;
+        }
+
+        /* Add a black background color to the top navigation */
+        .topnav {
+            background-color: #417690;
+            overflow: hidden;
+        }
+
+        /* Style the links inside the navigation bar */
+        .topnav a {
+            float: left;
+            color: #f5dd5d;
+            text-align: center;
+            padding: 14px 16px;
+            text-decoration: none;
+            font-size: 17px;
+        }
+
+        #dataset-manage {
+            display: block;
+        }
+
+        .datasets-table thead tr th {
+            padding-bottom: 5px;
+        }
+
+        .dataset-files-table-header-row th{
+            text-align: center;
+            padding-top: 5px;
+        }
+
+        .dataset-files-table-row th {
+            font-weight: normal;
+            text-align: left;
+            padding-left: 30px;
+            padding-bottom: 2px;
+        }
+
+        .dataset-files-table-col-2 {
+            text-align: right;
+        }
+
+        .mgr-tbl-dataset-header {
+            text-align: left;
+        }
+
+        .mgr-tbl-category-header {
+            text-align: left;
+        }
+
+        .mgr-tbl-content th {
+            font-weight: normal;
+            text-align: left;
+            padding-right: 15px;
+            padding-bottom: 2px;
+        }
+
+        #dataset-create {
+            display: none;
+        }
+
+    </style>
+</head>
+<body onload="initGuiViews()">
+<div id="container">
```

Comment:
This sort of boilerplate could be farmed out to a base template so you don't have to worry about all that here.